src/hooks/chatSidebar.ts

```diff
@@ -38,6 +42,13 @@ export const useChatSidebarControls = () => {
     }
   }, [expandSidebar, collapseSidebar, isExpanded]);
 
+  useMousetrap({
+    keys: KEYBOARD_SHORTCUTS.chat,
+    callback: toggleSidebar,
+    // TODO: bindRef: (null as never) as MutableRefObject<HTMLElement>,
+    withGlobalBind: true, // TODO: remove this once we have a ref to bind to
+  });
```

Comment:
@0xdevalias note that all uses of `useMousetrap`  are with `withGlobalBind: true`.  Should it be just be made default and be done until the use case comes up requiring RFin for `: false`?

Reply:
They are all global right now because non-global is broken. 

Changing the default based on that alone isn't a good reason to change the default.

Do you have other reason to believe that global would be a better default?

Reply:
It was just my 1c: unused default isn't a default really, just a hypothetical opportunity for an alternative behavior, which is now exposed in api, and as you said - broken. up to you to keep it, I am ok either way.

Reply:
Cool cool, let's leave it as is for now. Might help me feel the pain enough to find the time to fix it properly.

Reply:
ok, let's then consider resolved (will mark as such also in PR description)

Reply:
Sounds good.